### DISCUSSION TITLE
Can you write document whose name is the email address?

### DISCUSSION THREAD

**Discussion by seiay** [supersaintex/saint-sorting#43] [Q&A]
## 状況
 
 今PR（#38）上げてあるのと同じプログラムをつかって、clothing page からwriteしようとしたら、
`session_error: the email is not found.`
というエラーが出て、
authentication failed ...
となる。

## 試してほしいこと

自分の環境が悪いのかどうか知りたく、
同じエラーがでるかどうか確認してほしい。

**Comment by seiay** [comment]
sessionにemailをインサートした直後にgetするといけるけど、別の関数からgetしようとすると、今のエラーが出ることを確認してる。

**Reply by asana17** [reply]
これの再現手順を教えてもらえない？新たにブランチを作成してくれてもok

自分の環境ではwriteは可能だった。

**Reply by asana17** [reply]
sessionにinsertしている値が参照になっているのが原因な気がする。
とりあえず修正してみたから確認できる？https://github.com/supersaintex/saint-sorting/pull/38

**Reply by seiay** [reply]
> sessionにinsertしている値が参照になっているのが原因な気がする。 とりあえず修正してみたから確認できる？#38

同じエラー出てしまうな。
ブラウザに原因があるんかな。

**Reply by seiay** [reply]
> これの再現手順を教えてもらえない？新たにブランチを作成してくれてもok
> 
> 自分の環境ではwriteは可能だった。

新しくドラフトプルリク作成した（#44）
